plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimatorsFactory.kt

```diff
@@ -194,26 +185,21 @@ class CameraAnimatorsFactory internal constructor(mapDelegateProvider: MapDelega
   internal fun getMoveBy(
     offset: ScreenCoordinate
   ): Array<CameraAnimator<*>> {
-    val cameraOptions = mapCameraManagerDelegate.getCameraOptions(null)
+    val cameraState = mapCameraManagerDelegate.getCameraState()
     val centerTarget = CameraTransform.calculateLatLngMoveBy(
       offset,
-      cameraOptions,
+      cameraState,
       mapTransformDelegate,
       mapCameraManagerDelegate
     )
-    cameraOptions.center?.let { start ->
-      centerTarget.let { target ->
-        return Array(1) {
-          CameraCenterAnimator(
-            options = cameraAnimatorOptions(target) {
-              startValue(start)
-            },
-            block = defaultAnimationParameters[CameraAnimatorType.CENTER]
-          )
-        }
-      }
+    return Array(1) {
```

Comment:
not sure why this syntax is preferred over arrayOf? Because this creates a fixed size array?

Reply:
IDK 😄 I may change it but don't think it actually matters that much